build.xml

```diff
@@ -274,6 +274,20 @@
     </exec>
   </target>
 
+  <!-- ESLint jsDoc -->
```

Comment:
Maybe it's worth differentiating these comments (i.e. add "(test only)" to this first one and "(auto-fix eligible issues)" to the second).

Reply:
Sure!